test/test_helpers/configuration.rb

```diff
@@ -7,5 +7,5 @@
   config.redshift.port = 5432
   config.redshift.dbname = 'test'
   config.redshift.user = 'postgres'
-  config.redis.async = true
+  config.redis.async = ENV['CONFIG_REDIS_ASYNC'].to_s == 'true' ? true : false
```

Comment:
This can also be squashed.

Reply:
Yes, I wanted to keep the original commits of the PRs included in this integration branch so you could know what you reviewed already and what were your comments back then. Now that you've already reviewed this I can squash them to clean the history a bit.